airstack.go

```diff
@@ -0,0 +1,25 @@
+package main
```

Comment:
hey @jordipainan why there is a main package here?

Reply:
Because is located in the root folders, same as all other files.